src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -565,9 +574,14 @@ public void appendTransformHistory(final String key) {
      * @param newHistory list of new history strings to use
      */
     @Override
```

Comment:
My preference would be to remove the javadoc comment here so it doesn't override/conflict with the javadoc from the (IBDO) method it implements.  No sense trying to keep 2 copies in sync.